ptbr/tests/test_train_py.py

```
@@ -0,0 +1,140 @@
+from unittest.mock import patch
+
+ROOT = Path(__file__).resolve().parents[2]
+TRAIN_PY = ROOT / "train.py"
+_UNSET = object()
+
+
+class _DummyModel:
+    def __init__(self) -> None:
+        self.train_kwargs: dict | None = None
+
+    def to(self, dtype=None):
```

Ruff (0.15.1)

[warning] Unused method argument: `dtype`

(ARG002)